README.rst

```diff
@@ -34,19 +34,115 @@ adding the following features:
 * Support `drf-spectacular <https://github.com/tfranzel/drf-spectacular>`_ generated API schema
 * Additional fields and widgets (FileWidget, IntermediateM2MWidget, M2MField)
 
-Migration from django-import-export
------------------------------------
-Resources migration
-^^^^^^^^^^^^^^^^^^^
-Change ``Resource`` or ``ModelResource`` to
-``CeleryResource`` or ``CeleryModelResource`` respectively.
-
-Admin migration
-^^^^^^^^^^^^^^^
-Change ``ImportMixin``, ``ExportMixin``, ``ImportExportMixin``
-to ``CeleryImportMixin``, ``CeleryExportMixin`` or ``CeleryImportExportMixin`` respectively.
-
-License
--------
-* Free software: MIT license
+Usage
+-----
+
+To install django-import-export-extensions, run this command in your terminal:
+
+.. code-block:: console
+
+    $ pip install django-import-export-extensions
+
+Add `import_export_extensions` to INSTALLED_APPS
+
+.. code-block:: python
+
+    # settings.py
+    INSTALLED_APPS = (
+        ...
+        'import_export_extensions',
+    )
+
+Run `migrate` command to create ImportJob/ExportJob models and
+`collectstatic` to let Django collect package static files to use in the admin.
+
+.. code-block:: shell
+
+    $ python manage.py migrate
+    $ python manage.py collectstatic
+
+Prepare resource for your model
+
+.. code-block:: python
+
+    # apps/books/resources.py
+    from import_export_extensions.resources import CeleryModelResource
+
+    from .. import models
+
+
+    class BookResource(CeleryModelResource):
+        """Base resource for `Book` model."""
+
+        class Meta:
+            model = models.Book
+
+Use `CeleryImportExportMixin` class and set `resource_class` in admin model
+to import/export via Django Admin
+
+.. code-block:: python
+
+    # apps/books/admin.py
+    from django.contrib import admin
+
+    from import_export_extensions.admin import CeleryImportExportMixin
+
+    from .. import resources
+
+
+    @admin.register(models.Book)
+    class BookAdmin(CeleryImportExportMixin, admin.ModelAdmin):
+        """Admin UI for Book model."""
+        resource_class = resources.BookResource
+
+
+Prepare view sets to import/export via API
+
+.. code-block:: python
+
+    # apps/books/api/views.py
+    from ..resources import BookResource
+
+    from import_export_extensions.api import views
+
+
+    class BookExportViewSet(views.ExportJobViewSet):
+        """Simple ViewSet for exporting `Book` model."""
+        resource_class = BookResource
+
+
+    class BookImportViewSet(views.ImportJobViewSet):
+        """Simple ViewSet for importing `Book` model."""
+        resource_class = BookResource
+
+
+Don't forget to `configure Celery <https://docs.celeryq.dev/en/latest/getting-started/first-steps-with-celery.html>`_
+if you want to run import/export in background
+
+.. code-block:: python
```

Comment:
I think this code sample is redundant in docs of `django-import-export-extensions`.
Also, [celery docs](https://docs.celeryq.dev/en/stable/django/first-steps-with-django.html) provide another code samples for configuring it